src/IK/src/InverseKinematics.cpp

```diff
@@ -359,6 +366,7 @@ bool HumanIK::advance()
 {
     // Initialize ok flag to true
     bool ok{true};
+    // m_jointVelocityLimitsTask->update();
```

Comment:
is this comment useful?

Reply:
It is a leftover, thank you.

Reply:
removed with https://github.com/ami-iit/biomechanical-analysis-framework/pull/43/commits/b28b7e4667643905fbaea19371517e83e3fd9d60